src/js/controller/controls-loader.js

```diff
@@ -12,7 +12,7 @@ export function loadControls() {
             return ControlsModule;
         }, function() {
             controlsPromise = null;
-            chunkLoadErrorHandler();
+            chunkLoadErrorHandler(130)();
```

Comment:
remove extra `()`

Reply:
That's not extra, it's invoking the returned function